### PR TITLE
Allow billy to handle newly-defined Imago boundary sets

### PR DESCRIPTION
**Pull request by mileswwatkins**
`We shouldn't be string-parsing OCD IDs anyway.`
~the great @paultag

### CODE REVIEW

File: billy/web/api/handlers.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from piston.handler import BaseHandler, HandlerMetaClass
 
 AT_LARGE = ['At-Large', 'Chairman']
-ACTIVE_BOUNDARY_SETS = settings.BOUNDARY_SERVICE_SETS.split(",")
+ACTIVE_BOUNDARY_SETS_RE = settings.BOUNDARY_SERVICE_SETS
```
Comment (paultag) [inline billy/web/api/handlers.py]: Can we `re.compile` this?

File: billy/web/api/handlers.py [hunk 2]
```diff
@@ -490,7 +485,7 @@ def _ocd_id_to_shape_url(self, ocd_id):
         )
         data = json.load(urllib2.urlopen(url))
         geometries = filter(
-            lambda x: x['boundary_set']['name'] in ACTIVE_BOUNDARY_SETS,
+            lambda x: re.search(ACTIVE_BOUNDARY_SETS_RE, x['boundary_set']['name']),
```
Comment (paultag) [inline billy/web/api/handlers.py]: We should perhaps do a re.match  + handling for `sldl-nh-12-floterial` or something

Comment (mileswwatkins) [inline billy/web/api/handlers.py]: The `settings.py`'s RE already accounts for extra text afterwards, so that's all good, right?

Comment (mileswwatkins) [inline billy/web/api/handlers.py]: Uhh, never mind, apparently not in the version I pushed :)

**Comment by mileswwatkins** [comment]
Testing this locally now.

**Comment by mileswwatkins** [comment]
Confirmed working, including the Mississippi fix.

**Comment by mileswwatkins** [comment]
Hmm, except floterials aren't loading now. They're in the OCD response, though. Let me look into this some more, and I'll tie it up with you later.

**Comment by jamesturk** [comment]
@mileswwatkins I don't know if this is to stale to be useful to you, or if we should keep it around for later

**Comment by mileswwatkins** [comment]
Ideally when we turn on the OCD API, we'd also make a jump to supporting this newer, date-sensitive nomenclature of boundaries.

So, the question is, do we need Billy to also support these new-style boundaries? I guess so?

**Pull request closed without merging**